copy-sent-datetime-from-events.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import create_engine, and_
+from sqlalchemy.orm import sessionmaker
+from secure_message.repository.database import SecureMessage, Events
+from config import Config # NOQA
+
+engine = create_engine(Config.SQLALCHEMY_DATABASE_URI)
+Session = sessionmaker(bind=engine)
+session = Session()
+
+try:
+    messages_without_sent_at = session.query(SecureMessage).filter(SecureMessage.sent_at == None).all() # noqa
```

Comment:
There is  any idea of what could be or will be in a the future the size of this result ? query.all could get maybe dangerous in case of many data. There is any need of maybe "yeld_per" need ?

Reply:
That's a fair point, it's easy to forget the number of rows I might be dealing with...
I'll find out how many, and if it's a lot (and I suspect it is).  I'll add the yield_per